doc/examples/viz_bundles.py

```diff
@@ -170,8 +170,8 @@
 
 lengths = length(bundle_native)
 
-hue = [0.5, 0.5]  # red only
```

Comment:
I am confused by the comment here. This is the same as above, so how could they both be `red only`?

Reply:
Good catch! should be blue here